lib/wick-get-iface-ip

```diff
@@ -33,7 +33,17 @@ wickGetIfaceIp() {
         allInterfaces=true
     fi
 
-    if wickCommandExists ifconfig; then
+    # Works for systemd distros
+    if wickCommandExists networkctl; then
+        cmd=(networkctl status)
+
+	if [[ -n "$interface" ]]; then
+	    cmd[${#cmd[@]}]="$interface"
+	fi
+
+       # FIXME: Unfortunately, IPv4 only
+       list=( $("${cmd[@]}" 2>/dev/null | grep -oP "\b(?:[0-9]{1,3}\.){3}[0-9]{1,3}\b") )
```

Comment:
Does this `grep` work on mac?  Just asking because I don't know about the -o and -P options.